specification/metadata/metadata_generator.md

```diff
@@ -0,0 +1,37 @@
+# FOCUS Metadata Generator
+The FOCUS metadata provider logical section provides metadata about the provider of the FOCUS data. 
+
+## Logical Section: Schema Generator
+
+## Schema Definition Metadata
+
+* ### billing_generator
+    * use: Human readable name of the entity that is generating the billing data 
+    * type: STRING
+* ### provider_prefix
+    * use: Shortened name of the provider supplied to be used as the prefix for tags and may be used to modify the name of custom columns.
```

Comment:
What happens when there are multiple prefixes for different scenarios? I need to look at Tags but I recall internalizing scenarios where we would use different prefixes and not a single value.